jobs/job-scheduler/src/main/java/com/paladincloud/jobscheduler/service/JobScheduler.java

```diff
@@ -110,18 +113,24 @@ public void scheduleCollectorJobs() {
         try {
             ConfigUtil.setConfigProperties();
 			  azureEnabled=Boolean.parseBoolean(System.getProperty(AZURE_ENABLED));
-			  awsEnabled=Boolean.parseBoolean(System.getProperty(AWS_ENABLED)); 
-				if (awsEnabled) {
+			  awsEnabled=Boolean.parseBoolean(System.getProperty(AWS_ENABLED));
+				if (!wizEnabled && awsEnabled) {
```

Comment:
@arunpaladin @akashpaladin

why is Wiz flag hardcoded here? Since this is CE, we must not expose anything EE here.

Reply:
We have to find another way to run this, which should be driven by some config by either CE or EE

Reply:
sure @plyubich 
I have added configs